torch_xla/csrc/aten_cuda_functions.cpp

```diff
@@ -0,0 +1,22 @@
+#include "torch_xla/csrc/aten_cuda_functions.h"
+
+#include <pybind11/pybind11.h>
+
+#include <cassert>
+#include <stdexcept>
+
+static void fail(const char* name) {
+  throw std::runtime_error("PyTorch was compiled without CUDA support.");
+}
+
+namespace c10::cuda {
+
+c10::DeviceIndex current_device() { fail("c10::cuda::current_device()"); }
```

Comment:
This mean, if pytorch is built without CUDA, then these phony definition will be used. Without it, doing `import torch_xla` will fail with something like `ImportError: xla/_XLAC.cpython-310-x86_64-linux-gnu.so: undefined symbol: _ZN3c104cuda14current_deviceEv`?

Reply:
Exactly.